### DISCUSSION TITLE
Can I add some comment for each abbreviation?

### DISCUSSION THREAD

**Discussion by 1stzer** [olets/zsh-abbr#142] [General]
The `abbr` command is very nice.
I add many abbreviations.
Can `abbr list` show some comments for each abbreviation.

**Comment by olets** [comment]
Interesting, can you give an example?

**Comment by 1stzer** [comment]
I copy the file `user-abbreviations` to another named `user-abbreviations.ini` and add some comments in the copy.
Then I can use fzf to list all abbreviations and preview comments.
But when I modify a abbreviation, I have to modify my copy manually.😭.
So this very nice if I can add some comments in the file `user-abbreviations`. 

**Example below**:
```ini
# user-abbreviations.ini

# Upload all file to aliyun
"aliu"="aliyunpan upload -exn \"^\.\" \$ALIYUNPAN_ALL_DIR /"
```

```shell
fa() {
    sel=$(abbr list |
          fzf  --height 40% \
               --layout=reverse \
               --preview "bash $XDG_CONFIG_HOME/fzf/abbr_preview.sh {}" \
               --bind 'enter:become(echo {} | cut -f 2- -d = | cut -c 2- | rev | cut -c 2- | rev)')

    print -z -- "$(echo "${sel}" | sed -r 's#\\([0-9])#\\\\\1#g')"
}
```

<img width="1223" alt="image" src="https://github.com/olets/zsh-abbr/assets/25565773/41bb5f33-3ae8-4441-b0b4-8e9c5ec20372">
